capstone/pom.xml

```diff
@@ -34,9 +35,17 @@
     <dependency>
       <groupId>org.springframework.cloud</groupId>
       <artifactId>spring-cloud-gcp-starter-data-datastore</artifactId>
-      <version>1.2.3.RELEASE</version>
+      <version>${spring-cloud-gcp.version}</version>
```

Comment:
We might as well bring in the BOM to avoid specifying versions for Spring Cloud GCP deps.